src/pages/home/HeaderView.tsx

```diff
@@ -304,6 +307,7 @@ function HeaderView({report, parentReportAction, reportID, onNavigationMenuButto
                 )}
             </View>
             {!isLoading && canJoin && shouldUseNarrowLayout && <View style={[styles.ph5, styles.pb2]}>{joinButton}</View>}
+            {!isLoading && isChatUsedForOnboarding && shouldUseNarrowLayout && <View style={[styles.pb3, styles.ph5]}>{freeTrialButton}</View>}
```

Comment:
Excuse me @FitseTLT, could you clarify the purpose of displaying `freeTrialButton` here? Thank you.

Reply:
This is for the narrow layout case where we display it on a new line under the title unlike the large screen case.